lib/postprocessing/image.js

```diff
@@ -94,113 +93,50 @@ function processBasicImageOperations(img, instructions) {
         img.units('pixelsperinch');
         if (typeof instructions.convertToDpi === 'object') {
             const { xdpi, ydpi } = instructions.convertToDpi;
-            img = img.resample(xdpi, ydpi);
+            img.resample(xdpi, ydpi);
         } else {
-            img = img.resample(instructions.convertToDpi, instructions.convertToDpi);
+            img.resample(instructions.convertToDpi, instructions.convertToDpi);
         }
     }
-
-    console.log('Processing rendition... Done!');
-    return img;
```

Comment:
I'd like you to keep that, so we can have a fluent API (aka you can chain function calls, since they return the `img` object).

Reply:
We do not need it at all and it allows to make `img` in functions that define it as local var (not as argument) a const. We aren't able to chain our own functions that way...

Reply:
The main reason I removed it was because it removed a considerable amount of `img = ` from the code all over the place, making it easier to read. Also it clears up that this isn't required and makes things easier since you eg. can have `img` as a `const` instead of a `let`, which will surely be helpful in future code additions here.

Reply:
Well, I'd return `img` in all cases... If there is no operation done on `img`, the function would be a noop.